spec.md

```diff
@@ -0,0 +1,238 @@
+# Open Container Initiative
+
+## Artifacts Specification
+
+This specification defines using the [index][image-index] and [manifest][image-manifest] to define new artifact types, persisted within the [OCI Distribution Spec][distribution-spec]
+
+The goal of this specification is to standardize artifact distribution, leveraging the infrastructure used to run [OCI Distribution Spec][distribution-spec] compliant registries.
```

Comment:
do you really mean standardize artifact distribution here? 

This is more for the distribution spec.. artifacts spec should be for supporting artifact types in distribution and other types of apis...

Reply:
Please refer to updated text to see if this is addressed.